package.json

```diff
@@ -45,15 +49,17 @@
 	"dependencies": {
 		"arrify": "^2.0.1",
 		"cp-file": "^7.0.0",
-		"globby": "^9.2.0",
-		"has-glob": "^1.0.0",
+		"glob": "^7.1.6",
 		"junk": "^3.1.0",
+		"mkdirp": "^1.0.4",
```

Comment:
This is no longer needed. You can use the recursive option in `fs.mkdir`.

Reply:
Removed